pyramid_hypernova/types.py

```diff
@@ -9,6 +9,7 @@
 Job = namedtuple('Job', (
     'name',
     'data',
+    'context',
```

Comment:
it's not really clear what `context` is... if I ran across this in the code I'd probably think "oh, it's unused, let's delete it".

Can you maybe add  a comment/docstring explaining the purpose of `context`? Just saying (making this up) "`context` is an arbitrary object which is passed verbatim to the render function (?). It isn't used by pyramid_hypernova, but can be used by plugins to pass information along." or something like that would be a big help.